spring-cloud-zookeeper-discovery/src/main/java/org/springframework/cloud/zookeeper/discovery/ZookeeperDiscoveryClient.java

```diff
@@ -50,11 +51,11 @@ public String description() {
 
 	@Override
 	public org.springframework.cloud.client.ServiceInstance getLocalServiceInstance() {
-		ServiceInstance<ZookeeperInstance> serviceInstance = serviceDiscovery.getServiceInstance();
+		ServiceInstance<Object> serviceInstance = serviceDiscovery.getServiceInstance();
```

Comment:
What I was thinking of is that we had

`ServiceInstance<ZookeeperInstance>`

and now we have

`ServiceInstance<Object>`

I'm curious why did we go from `ZookeeperInstance` to `Object`?